tests/plugins/chart/basic_chart.test.ts

```diff
@@ -1527,24 +1527,29 @@ describe("Chart design configuration", () => {
     expect(chart.chartJsConfig.data!.labels).toEqual(["3/1/2022", "2022/03/02"]);
   });
 
+  function getTooltipItem(
+    value: number,
+    chartType: string,
+    chart: ChartConfiguration
```

Comment:
Why did you add `chart` as an argument, if it for doing nothing with it and returning it as is ?